app/view/panel/application/GeneralModel.js

```diff
@@ -17,7 +17,15 @@ Ext.define('MoMo.admin.view.panel.application.GeneralModel', {
                 'afterwards!',
         activeFieldLabel: 'Active',
         activeBoxToastText: '<b>Note:</b> This application will be ' +
-                'deactivated and will not be visibile to anyone afterwards!'
+                'deactivated and will not be visibile to anyone afterwards!',
+
+        appData: {
+            name: null,
+            description: null,
+            language: 'en',
```

Comment:
IMO, we need to think about the translate mechanism as soon as possible. The code at the moment is really dissimilar. Admin interface is completely in english, at the same time the most tools (but not all - e.g. RBMA) are in german. I've already opened  the https://github.com/terrestris/momo3-frontend/issues/30 issue.

I think we should get more harmonization as long as we don't have to much tools and classes to avoid the big refactoring afterwards.

This note doesn't really affect your code, it's just a remark for the (nearest) future :wink: